src/kubeseal_auto/secrets/creation.py

```
@@ -0,0 +1,143 @@
+        "--namespace",
+        secret_params.namespace,
+        _DRY_RUN_CLIENT,
+        "-o",
+        "yaml",
+        *entries,
+    ]
+
+    ic(cmd)
+
+    with open(output_path, "w") as f:
+        subprocess.run(cmd, stdout=f, check=True)
```

Ruff (0.14.7)

`subprocess` call: check for execution of untrusted input

(S603)